opfpush/src/main/java/org/onepf/opfpush/PushProvider.java

```diff
@@ -90,14 +90,14 @@
      * Callback method, that called when the application state change, like update to new version,
      * or system state changed, like update firmware to a newer version.
      * <p/>
-     * When this method call you registration is invalid
-     * and you need reset all saved registration data.
+     * If this method is called, your registration became invalid
```

Comment:
Not sure, but probably you wished to say
"If this method is called, your registration becomes invalid"